code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -140,6 +145,35 @@
 	node_machine.disconnect(src)
 	nodes[i] = null
 
+/**
+ * Called by nullify_node(), used to remove the pipeline the component is attached to
+ * Arguments:
+ * * -reference: the pipeline the component is attached to
+ */
+/obj/machinery/atmospherics/proc/nullify_pipenet(datum/pipeline/reference)
```

Comment:
Is this not inherently behavior of components? why is it here?

Reply:
same above